### PR TITLE
add new redirects

### PR DESCRIPTION
**Pull request by nathakits**
add 2 new redirects - line 72-73
format file

### CODE REVIEW

**Comment by papr** [comment]
@nathakits can you please include the redirects from #497?

**Comment by nathakits** [comment]
@papr do you still want me to include the redirects from #497 ?

**Comment by papr** [comment]
@nathakits Based on the PR title, I thought there would be a new redirect mechanism or something similar. In that case, it would have been nice to add the redirects from #497. But then I saw, that this PR was only formatting the existing redirect file without changing anything in the actual redirect mechanism. This is why I marked my comment as outdated.

File: src/.vuepress/enhanceApp.js [hunk 1]
```diff
@@ -56,26 +56,29 @@ export default ({ Vue, router, options }) => {
...
-  }
+    "/#export": "/core/software/pupil-player/#export",
+    "/#surface-tracking": "/core/software/pupil-capture/#surface-tracking",
+    "/#analysis-plugins": "/core/software/pupil-player/#analysis-plugins",
+    "/#pupil-capture": "/core/software/pupil-capture/",
+    "/#pupil-player": "/core/software/pupil-player/",
+    "/#network-plugins": "/core/software/pupil-capture/#network-plugins",
+    "/#interprocess-and-network-communication": "/developer/core/network-api/",
+    "/#annotations": "/core/software/pupil-capture/#annotations",
+    "/cloud/enrichments/#reference-image-mapper":
```
Comment (nathakits) [inline src/.vuepress/enhanceApp.js:72]: @papr there are 2 new redirects here, so should I include yours or add these to your PR instead?

Comment (papr) [reply src/.vuepress/enhanceApp.js:72]: Please move the redirects from #497 to yours. Then I can dedicate #497 to the software download links.

Comment (mikelgg93) [reply src/.vuepress/enhanceApp.js:72]: Shouldn't "/cloud/enrichments/#reference-image-mapper": redirect to "/invisible/explainers/enrichments/#reference-image-mapper" ? 

And if #550 comes alive, then to "/invisible/explainers/enrichments/reference-image-mapper"

**Comment by nathakits** [comment]
@papr I have added your redirects but removed `.html` from the URL string. There are some `.md` files that is missing the `permalink` in the frontmatter, so if you add the permalink in the markdown file then the URL will no longer have `.html`.

**Comment by papr** [comment]
@nathakits Ah, thank you!

**Comment by papr** [comment]
Started deployment to staging

**Comment by papr** [comment]
Server-side redirect from `/cloud` works as expected for me

**Review by papr — APPROVED** [review]

File: src/.vuepress/enhanceApp.js (hunk 1 above)
Comment (marc-tonsen) [reply src/.vuepress/enhanceApp.js:74]: @nathakits #550 has been merged just now, so the RIM link should be updated now to /invisible/explainers/enrichments/reference-image-mapper

**Comment by nathakits** [comment]
RIM Link updated

**Review by papr — APPROVED** [review]

**Comment by papr** [comment]
@nathakits Thank you for taking the time to update the redirects!

**Pull request merged**